ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -340,6 +413,7 @@ open class AvatarView: UIView {
 
     private var hasBorder: Bool = false
     private var hasCustomBorder: Bool = false
+    private var currentFallbackImageStyle: AvatarFallbackImageStyle?
```

Comment:
nit: would "fallbackImageStyle" be a simpler name? Not sure what makes it "current" more than any other property.